src/components/navbar/index.cjsx

```diff
@@ -93,6 +94,7 @@ module.exports = React.createClass
             title={<UserName/>}
             ref='navDropDown'>
             {menuItems}
+            <AdminConsole/>
```

Comment:
I don't think you need to create a whole `Component` just for the admin button. I think you can do something like the following at the top of `render`:

```coffee
if CurrentUserStore.isAdmin()
  adminLink = <BS.MenuItem
      href='admin'
      key='dropdown-item-admin'>Admin Console
    </BS.MenuItem>
```

and then replace this line with `{adminLink}`